examples/source/Basic_User_Consent_Flow/Basic_User_Consent_Flow.html

```diff
@@ -6,24 +6,15 @@
 teaserImage: '/static/samples/img/teaser/basic_user_consent_flow.jpg'
 
 --->
+
 <!--
   ## Introduction
 
-  Users today want additional control over their online experience. Additionally, publishers are
-  faced with a variety of different demands on how they provide notice and choice to their
-  users –  from vendor policies to evolving legal requirements. The open source AMP Project is working
-  to give publishers and tech vendors tools to implement their preferred user controls and to support their varied
-  individual compliance requirements on their AMP pages.
-
-  <p>This page demonstrates how a basic blocking consent flow can be built, which would just show a
-  simple blocking popup with an accept and reject button. On reject some content in the page will be blocked.</p>
-
-  [tip type="important"]
-    Important! The consent is saved to localstorage - after accepting/rejecting it once you will
-    only get the popup dialog again after deleting the localstorage content for this host,
-    for example via <a href="https://developers.google.com/web/tools/chrome-devtools/manage-data/local-storage#local-storage">
-    Chrome Dev Tools</a>.
-  [/tip]
+  Users today want additional control over their online experience. Additionally, publishers are faced with a variety of different demands on how they provide notice and choice to their users - from vendor policies to evolving legal requirements. The open source AMP Project is working to give publishers and tech vendors tools to implement their preferred user controls and to support their varied individual compliance requirements on their AMP pages.
+
+  This page demonstrates how a basic blocking consent flow can be built, which would just show a simple blocking popup with an accept and reject button. On reject some content in the page will be blocked.
+
+  Important: The consent is saved to localstorage - after accepting/rejecting it once you will only get the popup dialog again after deleting the localstorage content for this host, for example via [Chrome Dev Tools](https://developers.google.com/web/tools/chrome-devtools/manage-data/local-storage#local-storage).
```

Comment:
Are callouts still not supported?

Reply:
Both types actually, so having a one-liner prefixed with `Important:` has the same effect as `[tip type="important"]`. 🙂